src/OpenTK.GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -24,128 +24,185 @@ internal interface IGLFW
+        /// </para>
+        /// <para>
         /// This function must only be called from the main thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.InvalidEnum"/> and <see cref="ErrorCode.InvalidValue"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.InvalidEnum"/> and <see cref="ErrorCode.InvalidValue"/>.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="Init"/>
         void InitHint(InitHint hint, bool value);
 
         /// <summary>
+        /// <para>
         /// This function retrieves the major, minor and revision numbers of the GLFW library.
         /// It is intended for when you are using GLFW
         /// as a shared library and want to ensure that you are using the minimum required version.
+        /// </para>
+        /// <para>
         /// Any or all of the version arguments may be <c>out _</c>.
+        /// </para>
         /// </summary>
         /// <param name="major">Where to store the major version number, or <c>out _</c>.</param>
         /// <param name="minor">Where to store the minor version number, or <c>out _</c>.</param>
         /// <param name="revision">Where to store the revision number, or <c>out _</c>.</param>
         /// <remarks>
+        /// <para>
         /// This function may be called before <see cref="Init"/>.
-        ///
+        /// </para>
+        /// <para>
         /// This function may be called from any thread.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="GetVersionString"/>
         void GetVersion(out int major, out int minor, out int revision);
 
         /// <summary>
+        /// <para>
         /// This function returns the compile-time generated version string of the GLFW library binary.
         /// It describes the version, platform, compiler and any platform-specific compile-time options.
         /// It should not be confused with the OpenGL or OpenGL ES version string, queried with <c>glGetString</c>.
-        ///
+        /// </para>
+        /// <para>
         /// Do not use the version string to parse the GLFW library version.
         /// The <see cref="GetVersion"/> function provides the version of the running library binary in numerical format.
+        /// </para>
         /// </summary>
-        /// <returns>The ASCII encoded GLFW version string.</returns>
+        /// <returns>The ASCII-encoded GLFW version string.</returns>
         /// <remarks>
+        /// <para>
         /// This function may be called before <see cref="Init"/>.
-        ///
+        /// </para>
+        /// <para>
         /// The returned string is static and compile-time generated.
-        ///
+        /// </para>
+        /// <para>
         /// This function may be called from any thread.
+        /// </para>
         /// </remarks>
         /// <seealso cref="GetVersion"/>
-        string GetVersionString(); // TODO: use marshalling to ASCII C string
```

Comment:
does ADL already automatically marshall ASCII c strings? otherwise it would be good if there was a comment to make us remember that something is missing...

Reply:
I don't know if it does/doesn't, will have to ask Jax.